Suggestions

Dingenis opened the pull request:
Hi,

Super cool library! I think it is written really well, I do have some suggestions :) 
Looking forward to what you think.

Best regards,

Bram

DanielHuisman commented:
Could you please explain why you want to make these changes? I can't exactly tell what your goal is for this PR.

Dingenis commented:
Sure! My goal was to make my first open source contribution by some small additions. Sorry if I came across a little to enthusiastic in creating a issue for it. 😅 

Small overview of what I changed with each commit. 
53ff29fb0fbbc27457bd56691de21506fc4b3c62 : Added another Tikkie API library.
60b8dae7bb5131ac80be63eddd2b8aacab52c34e : Made the load private key function into a promise one.
ebaa7e9e1190f333b4b53aa3dbc0ee22a45f1ed3 : Refactored some functions by splitting them out so it becomes easier to maintain and understand.
c17d5e55ab25a36933818ef2026e261613aa6337 : Changed the order of functions to make a more logical story of code.
